src/parcels/_datasets/structured/generic.py

```diff
@@ -225,3 +247,54 @@ def _unrolled_cone_curvilinear_grid():
     ),
     "2d_left_unrolled_cone": _unrolled_cone_curvilinear_grid(),
 }
+
+_COMODO_TO_2D_SGRID = {  # Note "2D SGRID" here is meant in the context of Grid2DMetadata and SGRID convention (i.e., 1D depth)
+    "XG": "node_dimension1",
+    "YG": "node_dimension2",
+    "XC": "face_dimension1",
+    "YC": "face_dimension2",
+    "ZG": "vertical_dimensions_dim1",
+    "ZC": "vertical_dimensions_dim2",
+}
+datasets_sgrid = {
```

Comment:
I thought that it was better to create a separate dict of datasets here. We could have attached the grid on the original datasets, but this would have conflicted with xgcm internals as I don't think its robust to having both conventions defined.

And this way we can provide datasets from the "comodo way of thinking" or the "SGRID way of thinking"